src/licensedcode/data/rules/biosl-4.0_1.yml

```diff
@@ -3,9 +3,9 @@ is_license_notice: yes
 relevance: 100
 notes: seen in https://raw.githubusercontent.com/davidtiefenthaler/BOPT-DemandForecast/main/README.md
 ignorable_copyrights:
-    - Copyright (c) 2009, 2018 Robert Bosch GmbH
+    - Copyright (c) 2009, 2018 Robert Bosch GmbH and its subsidiaries
```

Comment:
I already fixed this in a branch here https://github.com/nexB/scancode-toolkit/commit/f69cd894061bfd1609e46f10f0c0805dc7bdf061 so it might not be needed here

Reply:
@pombredanne Thanks!
 I added this because of some failing check. Shall I remove this commit?

Reply:
That's OK ... let's keep this, I will merge.